autoemulate/simulations/advection_diffusion.py

```diff
@@ -0,0 +1,243 @@
+import numpy as np
+import scipy.sparse as sp
+import torch
+from scipy.fft import fft2, ifft2
+from scipy.integrate import solve_ivp
+
+from autoemulate.core.types import NumpyLike, TensorLike
+from autoemulate.simulations.base import Simulator
+
+integrator_keywords = {}
+integrator_keywords["rtol"] = 1e-6
+integrator_keywords["method"] = "RK45"
+integrator_keywords["atol"] = 1e-8
+
+
+class AdvectionDiffusion(Simulator):
```

Comment:
I think it will be interesting to see the the predictive performance of AE with this system. Perhaps we should open an issue to add to the dim reduction tutorial? Or a new tutorial if different features of AE could be useful to demonstrate on this dataset (e.g. a different transform).